docs/WritingTests.asciidoc

```diff
@@ -261,6 +261,19 @@ the clone source job or specifying the variable explicitly:
 clone_job.pl --from https://openqa.opensuse.org  --host localhost 24 TESTDEBUG=1 SKIPTO=consoletest-yast2_i
 ----
 
+==== Performance impact
+
+Keep in mind that creating VM snapshots can induce quite high I/O load. This
+should not impact the stability of openQA jobs but can increase job execution
+time. If you run jobs on a machine where responsiveness of other services
+matter, for example your desktop machine, consider executing the worker
+services with +ionice+ to reduce the risk of your system becoming
```

Comment:
No, don't do that - patch the systemd service instead:

https://www.freedesktop.org/software/systemd/man/systemd.directives.html#IOSchedulingPriority=

Reply:
fixed